### PR TITLE
chore(RAMLvalidation): add latest marathon snapshot

### PR DESCRIPTION
**Pull request by Poltergeist**
This introduces the latest marathon snapshot

### CODE REVIEW

**Comment by Poltergeist** [comment]
alfred retest this

**Comment by nLight** [comment]
```
# Downloading marathon 1.6.0-pre-218-g36ad9bd archive
# Installing raml files
cp: /var/folders/47/pm84z4sj4ts764ffzg3vlrkr0000gp/T/tmp.SO3JctqO/marathon-docs-1.6.0-pre-218-g36ad9bd/docs/rest-api/public/api/v2/schema: No such file or directory
cp: /var/folders/47/pm84z4sj4ts764ffzg3vlrkr0000gp/T/tmp.SO3JctqO/marathon-docs-1.6.0-pre-218-g36ad9bd/docs/rest-api/public/api/v2/types: No such file or directory
cp: /var/folders/47/pm84z4sj4ts764ffzg3vlrkr0000gp/T/tmp.SO3JctqO/marathon-docs-1.6.0-pre-218-g36ad9bd/docs/rest-api/public/api/v2/*.raml: No such file or directory
! Updated RAML in /Users/dmitriy/workspace/dcos-ui-master/dcos-ui/src/resources/raml/marathon/v2
```

and then `/Users/dmitriy/workspace/dcos-ui-master/dcos-ui/src/resources/raml/marathon/v2` is empty

File: scripts/utils/marathon [hunk 1]
```diff
@@ -29,17 +29,16 @@ function install_marathon_raml() {
 
   # Download and expand on-the-fly the marathon tarball into a temporary folder
   header "Downloading marathon ${VERSION} archive"
-  local URL="https://downloads.mesosphere.com/marathon/snapshots/marathon-raml-${VERSION}.tgz"
+  local URL="https://downloads.mesosphere.com/marathon/builds/marathon-docs-${VERSION}.tgz"
   local TMP_DIR=`mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir'`
-  curl -L ${URL} | tar -zx -C $TMP_DIR
```
Comment (nLight) [inline scripts/utils/marathon]: I bet we want to keep this line 🤔

**Comment by Poltergeist** [comment]
Yes we want to keep the line I must have added this line by accident to the commit thanks for investigating.

**Comment by orlandohohmeier** [comment]
L👀ks good!

**Pull request merged**